pages/travel-and-leave/travel-and-leave-policies/first-time-travel-get-in-concur-pre-olu.md

```diff
@@ -1,21 +1,27 @@
 ---
-title:
-  First-time travel guide - Step 1 - Get set up to travel for TTS
-  <em>before</em> GSA onboarding
+title: Get set up to travel for TTS <em>before</em> GSA onboarding
+subtitle: "First-time travel guide: Step 1a"
 questions:
   - travel
   - tts-travel@gsa.gov
   - text: Book office hours
     url: https://sites.google.com/a/gsa.gov/tts-office-hours/
 redirect_from:
   - /first-time-travel-get-in-concur-pre-olu/
-current_step: 1
+step_indicator:
+  current_step: 1
+  substep: 1a
+  hide_steps: true
 sidebar:
   kind: travel
   current: 1. First time setup
 ---
 
-{% include "step-indicator.html" steps: travel-steps.first_time_steps current_step:current_step  %}
+{% include "step-indicator.html"
+steps: travel-steps.first_time_steps
+current_step: step_indicator.current_step
+substep: step_indicator.substep
+hide_steps: step_indicator.hide_steps %}
 
 Steps to take after onboarding:
```

Comment:
suggest moving this content to the end of the page, so it's clear that these are steps for after the actions listed.

Reply:
I ended up removing it, since the process described seemed to reference these pages.